### DISCUSSION TITLE
Cant download

### DISCUSSION THREAD

**Discussion by MikeDee101** [ImageMagick/ImageMagick#3516] [Help]
So everytime i try downloading imagemagick i end up with this error, My internet is okay, tried different devices, same issue. ![image](https://user-images.githubusercontent.com/29946953/113949309-9b2cb400-9817-11eb-9516-503c79407b6c.png)
Anyone with a workaround?

**Comment by fmw42** [comment]
Binaries can be download from https://imagemagick.org/script/download.php.  The source code download page seems to be offline or they are upgrading the directory with new software. Please give it some time. Or try https://imagemagick.org/script/install-source.php
